liquidprompt

```diff
@@ -1550,6 +1556,70 @@ _lp_http_proxy_color() {
     lp_http_proxy_color="${LP_COLOR_PROXY}${LP_MARK_PROXY}${NO_COL}"
 }
 
+_lp_env_vars() { # [color_set, [color_unset]]
+    # Expects LP_ENV_VARS to be an array containing items of the form:
+    # "<ENV_VAR_NAME> <string if set>[ <string if unset>]"
+    # Strings may be `%s`, which will be replaced by the variable's actual content.
+
+    (( LP_ENABLE_ENV_VARS )) || return 2
+    [[ -n "${LP_ENV_VARS-}" && ${#LP_ENV_VARS} -gt 0 ]] || return 1
+
+    local color_set="${1-}"
+    local color_unset="${2-}"
+    local color_no="${NO_COL}"
+    if [[ -z "$color_set" && -z "$color_unset" ]]; then
+        color_no=""
+    fi
+
+    local var_rep var evar fmt def
+
+    lp_env_vars=()
+    # For all user-defined setup.
+    for var_rep in ${LP_ENV_VARS[@]+"${LP_ENV_VARS[@]}"}; do
+        IFS=' ' read -r var fmt def <<<"$var_rep"
+        # Variable name and format has to be set, but default is optional.
+        if [[ -n "${var}" && -n "${fmt}" ]]; then
+            # A way to test for undefined ${!var}
+            if declare -xp "${var}" &>/dev/null; then
+                # Expands the underlying variable name.
+                if (( _LP_SHELL_zsh )); then
+                    # From https://www.shellcheck.net/wiki/SC2296 :
+                    # "Some Zsh specific parameter expansions like ${(q)value} trigger this warning,
+                    # but ShellCheck does not support Zsh."
+                    # shellcheck disable=SC2296
+                    evar="${(P)var}"
+                else
+                    # NOTE: indirection expansion are deprecated starting at bash 4.3, should use nameref.
+                    evar="${!var}"
+                fi
+                # Print tag/content or default.
+                if [[ -n "${evar}" && "${evar}" != "0" ]]; then
```

Comment:
Why skip this if the variable contents are `0`? Is that not a valid value that a user might want to see?

I see you have test coverage for it, but I need to know the reasoning, and so does the documentation.

Reply:
Usually, if, say, `VERBOSE=0`, then the feature is OFF, and I would not want to see it. But maybe it would be better to show the zero, indeed. The user can still `unset`.